src/components/contactSection.jsx

```diff
@@ -1,4 +1,29 @@
-const ContactSection = () => {
+export default function ContactSection() {
+  const onSubmit = async (event) => {
+      event.preventDefault();
+      const form = event.target
+      const formData = new FormData(form);
+  
+      formData.append("access_key", "a117415c-03e2-4afc-9726-c4ca8952dc76");
```

Comment:
ewe need a way to move this to secret key. Anyone can send a message to us, it should not be so. we might also need to regenerate these keys as they are now compromised.